great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -37,6 +44,52 @@ def __init__(
             data_context=data_context,
         )
 
+    def plot(self, data_assistant_result: DataAssistantResult):
```

Comment:
@NathanFarmer If at all possible, I would hope that we can move all of Plotting/Altair dependencies into the `DataAssistantResult` class.  I am hoping that this is possible, due to the fact that all fully-qualified parameter name, pertaining to metrics, have this form: `"$parameter.table_row_count.attributed_value"` -- and we have the constants (in `parameter_container.py` and exported by `__init__.py` therein) that will allow us to parse the middle (`table.row_count`) -- the metric name -- from it in a reliable way.  Please consider it (happy to discuss).  Thanks!

Reply:
@alexsherstinsky I think we can move most of the components except the `VolumeDataAssistant`-specific `Chart`s should remain in `volume_data_assistant.py`.